### PR TITLE
Pierogi

### PR DESCRIPTION
**Pull request by ghost**
Just trying to learn how to make PRs. Compiled and spawned it on a local serb, seems to work.

### CODE REVIEW

**Comment by Arthurdentist** [comment]
can you show us the sprite?

Also you need to add a recipe for this, otherwise its just admin spawn only.

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
firefox_2018-10-13_20-06-15

**Comment by MadmanMartian** [comment]
👍

**Comment by kilozombie** [comment]
You could make it spawn with actual cream in, and probably some more nutriment
I know that pierogi fills me the fuck up

**Comment by RealestEstate** [comment]
cream in pierogi would upset the meta and have heavy balance implications. I need a collab to put the balance and discussion tags on this so we can treat our new coder to the wonders of Github PR purgatory and quality discussion.

edit. I'm kidding.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -5696,6 +5696,15 @@ obj/item/weapon/reagent_containers/food/snacks/butteredtoast
 	..()
 	reagents.add_reagent(NUTRIMENT, 2)
 
+obj/item/weapon/reagent_containers/food/snacks/pierogi
```
Comment (DamianX) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: You forgot a `/` before `obj`

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -5696,6 +5696,15 @@ obj/item/weapon/reagent_containers/food/snacks/butteredtoast
...
+	name = "pierogi"
+	desc = "Dumplings with potatoes and sour cream inside"
```
Comment (DamianX) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Missing a period at the end

**Comment by Shadowmech88** [comment]
Would using a piece of flattened dough instead of flour be slightly more realistic?
Not that it really matters either way, but since we have flattened dough in the code already...

**Comment by SkowronX** [comment]
tak

**Comment by Hoshi-chan** [comment]
Hell yes

**Pull request merged**